tailwind.config.js

```diff
@@ -170,6 +170,11 @@ module.exports = {
           '0%': { opacity: 0 },
           '100%': { opacity: 1 }
         },
+        opacityInfinity: {
+          '0%': { opacity: 0 },
+          '50%': { opacity: 1 },
+          '100%': { opacity: 0 }
```

Comment:
Definitely need a phase where the elements remain invisible for a while
```suggestion
        opacityInfinity: {
          '0%': { opacity: 0 },
          '33%': { opacity: 1 },
          '66%': { opacity: 0 },
          '100%': { opacity: 0 }
```